### PR TITLE
feat(cli): add ReactRefreshPluginOptions receive

### PR DESCRIPTION
**Pull request by myNameIsDu**
add ReactRefreshPluginOptions receive

### CODE REVIEW

File: packages/cli/bin/ak-dev.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ program.addHelpText('after', () => {
   $ ak dev -p 8888
 
   ${chalk.gray('# run ak dev server without HMR')}
-  $ ak dev --no-hotReplace
+  $ ak dev --no-hot-replace
```
Comment (bigbigDreamer) [inline packages/cli/bin/ak-dev.js:52]:
```suggestion
  $ ak dev --no-hotReplace
```
look this line https://github.com/myNameIsDu/aktiv/pull/76/files#diff-ff0097cf4bb6fc14400937827586e93b92767108ee38e81b6ace045faf33b6ffL180

**Review by bigbigDreamer — COMMENTED** [review]
check it and confirm

File: packages/cli/bin/ak-dev.js (hunk 1 above)
Comment (myNameIsDu) [reply packages/cli/bin/ak-dev.js:52]: yeah

Comment (myNameIsDu) [reply packages/cli/bin/ak-dev.js:52]: so i want to use "hot-replease"

Comment (myNameIsDu) [reply packages/cli/bin/ak-dev.js:52]: how do you think?

Comment (bigbigDreamer) [reply packages/cli/bin/ak-dev.js:52]: image

**Review by bigbigDreamer — APPROVED** [review]
good job

**Pull request merged**